TicketSystem/TicketSystem.BLL/Services/TicketService.cs

```diff
@@ -49,7 +49,7 @@ public async Task<Ticket> GetTicketById(int id, CancellationToken cancellationTo
 
     public async Task<IEnumerable<Ticket>> GetTickets(CancellationToken cancellationToken)
     {
-        var ticketsEntity = await _ticketRepository.GetWithInclude(cancellationToken, false, null, null,
+        var ticketsEntity = await _ticketRepository.GetWithInclude(cancellationToken, null, null,
```

Comment:
Can we make null values as optional parameters in the method?

Reply:
![image](https://user-images.githubusercontent.com/92907361/219618124-216656c7-cff1-4dee-89d1-1ac285bffa66.png)
these are optional
but at the end there are params includeProperties, so it is necessary to specify them when calling the method

Reply:
Okay, it's super generic :D